hibike/devices/YogiBear/YogiBear.cpp

```diff
@@ -253,13 +224,22 @@ float readPWMInput() {
   return pwmInput;
 }
 
+void setPWMInput(float pwm) {
+	pwmInput = pwm;
```

Comment:
The only place pwmInput is actually changed is in motorDisable. We reset it to 0. Since this is the only place it is ever changed and the only scenario it should every be changed, I changed the function to be resetPWMInput() instead.